playwright/tina-playwright/tests/rich-text-editor/bold-parsing.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test } from "@playwright/test";
+import { checkMarkdownOutput } from "./markdown-parsing-utils";
+
+test.describe("should be able to parse bold from markdown", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("http://localhost:3000/admin/index.html#/~", {
+      waitUntil: "domcontentloaded",
+    });
+
+    page.click('button[data-test="enter-edit-mode"]');
+  });
+
+  test("Bold is correctly rendered from markdown using **", async ({
+    page,
+  }) => {
+    await checkMarkdownOutput(
+      page,
+      "Testing **Bold** rendering correctly in markdown",
+      "Bold",
+      "strong"
+    );
+  });
+
+  test("Bold is correctly rendered from markdown using __", async ({
+    page,
+  }) => {
+    await checkMarkdownOutput(
+      page,
+      "Testing __Bold__ rendering correctly in markdown",
+      "Bold",
+      "strong"
+    );
+  });
+});
```

Comment:
So this test should be failing currently 👀

Reply:
Why ?

Reply:
![2024-10-24 15 42 42](https://github.com/user-attachments/assets/1f01d50d-6100-4efc-b70c-769a467bde3d)

Doesn't seem to be working

Reply:
the __ get parse into \_\_ when saving in markdown, which is not part of the test (perhaps it is something we need to fix in future ?). but the test is for parsing markdown that why it works (md parse into html)